modules/otel-collector-components.adoc

```diff
@@ -740,6 +740,46 @@ The Kafka exporter exports logs, metrics, and traces to Kafka. This exporter use
 
 Connectors connect two pipelines.
 
+[id="forward-connector_{context}"]
+=== Forward connector
+
+The forward connector merges two pipelines of the same type.
+
+.OpenTelemetry Collector custom resource with an enabled forward connector
+[source,yaml]
+----
+receivers:
+  otlp:
+    protocols:
+      grpc:
+  jaeger:
+    protocols:
+      grpc:
+processors:
+  batch:
+exporters:
+  otlp:
+    endpoint: tempo-ingester:4317
```

Comment:
We ingest the traces to distributor. We can change it to tempo-simplest-distributor:4317 in this docs example.

Reply:
Fixed. Thank you.